app-crypt/libu2f_host/libu2f_host-1.1.10.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Yubico Universal 2nd Factor (U2F) Host C Library"
+DESCRIPTION="libu2fhost provides a C library and command-line tool \
+that implements the host-side of the U2F protocol.
+There are APIs to talk to a U2F device and perform the \
+U2F Register and U2F Authenticate operations."
+HOMEPAGE="https://developers.yubico.com/libu2f-host/
+	https://github.com/Yubico/libu2f-host"
+COPYRIGHT="2013-2014 Yubico AB"
+LICENSE="GNU GPL v3
+	GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/Yubico/libu2f-host/archive/libu2f-host-$portVersion.tar.gz"
+CHECKSUM_SHA256="45937c6c04349f865d9f047d3a68cc50ea24e9085d18ac2c7d31fa38eb749303"
+SOURCE_DIR="libu2f-host-libu2f-host-$portVersion"
+PATCHES="libu2f_host-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="0.1.10"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libu2f_host$secondaryArchSuffix = $portVersion
+	cmd:u2f_host$secondaryArchSuffix = $portVersion
+	lib:libu2f_host$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libjson_c$secondaryArchSuffix
+	lib:libhidapi$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libu2f_host_devel$secondaryArchSuffix = $portVersion
+	devel:libu2f_host$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	libu2f_host$secondaryArchSuffix == $portVersion base
+	lib:libjson_c$secondaryArchSuffix
+	lib:libhidapi$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libjson_c$secondaryArchSuffix
+	devel:libhidapi$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:gengetopt
+	cmd:help2man
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config
+	"
+
+BUILD()
+{
+	autoreconf -vfi
+	runConfigure ./configure
```

Comment:
We should add a commandBinDir here as it's  not working for x86_gcc2 $binDir refers to $prefix/bin/x86 on secondary architecture where it should be $prefix/bin, that would mean we only need "cmd:u2f_host" without $secondaryArchSuffix